### PR TITLE
document add order

### PR DESCRIPTION
**Pull request by ghost**
## Description
Docs Create order.
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: docs/api-references/order.md [hunk 1]
```diff
@@ -183,3 +183,51 @@ Show order detail
...
+        "id": 26,        
+        "user_id": 1,
+        "total": 6,
+        "order_details": [
+            {
+                "id": 18,
+                "product_id": 1,
+                "order_id": 26,
+                "quantity": 3,
+                "product_price": 2,
```
Comment (at-dungle) [inline docs/api-references/order.md]: sửa lại price thôi

File: docs/api-references/order.md [hunk 2]
```diff
@@ -183,3 +183,61 @@ Show order detail
...
+{
+    "meta": {
+        "status": "Successfully",
+        "code": 200
+    },
+    "data": {
+        "id": 26,        
+        "user_id": 1,
+        "total": 6,
+        "status": 1,
```
Comment (at-tuanphan) [inline docs/api-references/order.md]: status em trả về lun string lun xuống dưới js khỏi phải handle em

File: docs/api-references/order.md [hunk 3]
```diff
@@ -183,3 +183,61 @@ Show order detail
...
+|---|---|---|
+| user_id | string | Id Of User |
+| Total | number | total order |
+
+#### Response - Success
+```json
+{
+    "meta": {
+        "status": "Successfully",
+        "code": 200
```
Comment (at-tuanphan) [inline docs/api-references/order.md]: cái này trả về 201 vì là tạo mới em:
201 => created

**Review by ghost — COMMENTED** [review]
please check my comments

File: app/Order.php [hunk 4]
```diff
@@ -26,6 +26,30 @@ class Order extends Model
...
+     * @var array
+     */
+    protected $appends = ['status_order'];
+
+    /**
+     * Get the order's status.
+     *
+     * @return string
+     */
+    public function getStatusOrderAttribute()
```
Comment (at-tuanphan) [inline app/Order.php]: getStatusAttribute($status) {
return $this->statusOrder[$status];
}

update o apidoc lun em

**Review by ghost — COMMENTED** [review]
check my comment

**Review by ghost — APPROVED** [review]

File: app/Order.php [hunk 5]
```diff
@@ -26,6 +26,23 @@ class Order extends Model
         'status',
     ];
 
+    protected $statusOrder = [
+        1 => 'pendding',
```
Comment (at-tuanphan) [inline app/Order.php]: self::PENDING => 'pendding'
update het cho nhung cai kia di em

**Review by ghost — COMMENTED** [review]
please check my comment

**Review by ghost — APPROVED** [review]
LGTM [:+1:](http://lgtm.in/u/0aNBSIYwn)
[LGTM](http://lgtm.in/i/0aNBSIYwn)

**Pull request merged**